app/client/package.json

```diff
@@ -38,61 +38,63 @@
   "devDependencies": {
     "@cypress/instrument-cra": "1.4.0",
     "@esri/arcgis-rest-types": "3.7.0",
-    "@testing-library/jest-dom": "6.3.0",
-    "@testing-library/react": "14.1.2",
+    "@testing-library/jest-dom": "6.4.6",
+    "@testing-library/react": "16.0.0",
     "@testing-library/user-event": "14.5.2",
     "@types/file-saver": "2.0.7",
-    "@types/jest": "29.5.11",
-    "@types/node": "20.11.10",
+    "@types/jest": "29.5.12",
+    "@types/node": "20.14.5",
     "@types/papaparse": "5.3.14",
-    "@types/react": "18.2.48",
-    "@types/react-dom": "18.2.18",
+    "@types/react": "18.3.3",
+    "@types/react-dom": "18.3.0",
     "@types/react-ranger": "2.0.4",
-    "@types/smoothscroll-polyfill": "0.3.3",
+    "@types/smoothscroll-polyfill": "0.3.4",
     "@types/uuid": "9.0.8",
+    "ajv": "8.16.0",
```

Comment:
Do we use this anywhere directly, or was one of the other dependencies not declaring it as a dependency?

Reply:
It wouldn't run locally. I don't remember what the exact error was, but this was the suggested fix.